### PR TITLE
DDF-5423 make and use constants for common cql strings

### PR DESCRIPTION
**Pull request by zta6**
#### What does this PR do?
anyText ILIKE '%', and anyText ILIKE '*' are used in several places as defaults/fallbacks. This PR makes constants for those strings and uses those constants in a few places across the codebase. This constants will also be of use downstream.

#### Who is reviewing it? 

@andrewzimmer 
@hayleynorton 
@willwill96 

#### Select relevant component teams: 
@codice/ui 

#### What are the relevant tickets?
Fixes: #5423 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/js/cql.js [hunk 1]
```diff
@@ -19,6 +19,9 @@
 // jshint ignore: start
 const moment = require('moment')
 
+const ANYTEXT_WILDCARD = '"anyText" ILIKE \'%\''
+const ANYTEXT_EMPTY = '"anyText" ILIKE \'\''
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/js/cql.js]: :question: Should empty return different results than wildcard?

Comment (zta6) [reply ui/packages/catalog-ui-search/src/main/webapp/js/cql.js]: I'm not positive. I was told there was a difference between empty and wildcard, but I can play around a bit and see if that's true.

File: ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ import query from '../../react-component/utils/query'
 const fetchSuggestions = memoize(async attr => {
   const json = await query({
     count: 0,
-    cql: "anyText ILIKE ''",
+    cql: cql.ANYTEXT_EMPTY,
```
Comment (zta6) [inline ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js]: change to wildcard, as per @willwill96

**Comment by zta6** [comment]
build now

**Comment by willwill96** [comment]
Hero started

**Comment by willwill96** [comment]
### Hero Successful

**Comment by bdeining** [comment]
build now

**Pull request merged**